command/file.go

```diff
@@ -33,24 +33,28 @@ func CopyDir(src, dst string, isOnlyContent bool) error {
 }
 
 // RemoveDir ...
-func RemoveDir(dirPth string) error {
-	if exist, err := pathutil.IsPathExists(dirPth); err != nil {
-		return err
-	} else if exist {
-		if err := os.RemoveAll(dirPth); err != nil {
+func RemoveDir(dirPths ...string) error {
+	for _, dirPth := range dirPths {
+		if exist, err := pathutil.IsPathExists(dirPth); err != nil {
 			return err
+		} else if exist {
+			if err := os.RemoveAll(dirPth); err != nil {
+				return err
+			}
 		}
 	}
 	return nil
 }
 
 // RemoveFile ...
-func RemoveFile(pth string) error {
-	if exist, err := pathutil.IsPathExists(pth); err != nil {
-		return err
-	} else if exist {
-		if err := os.Remove(pth); err != nil {
+func RemoveFile(pths ...string) error {
```

Comment:
please use os.RemoveAll instead, to have an exact controlled behaviour (with Remove if you read the comment it is not exact what and how will happen) and also this function should not fail if there is something not existing as it will remove anyway

Reply:
if you will have the same logic for both function then please use shared codebase

Reply:
this function like:
```
func removeAll(pths ...string) error {
	for _, pth := range pths {
		if err := os.RemoveAll(pth); err != nil {
			return err
		}
	}
	return nil
}
```

is not used on so many places, why is it required to move out to a package?

Reply:
This is a utility method, which should be available to be used in every package.